libraries/stdlib/common/src/generated/_Strings.kt

```diff
@@ -139,7 +139,10 @@ public inline fun CharSequence.indexOfLast(predicate: (Char) -> Boolean): Int {
 
 /**
  * Returns the last character.
+ *
  * @throws [NoSuchElementException] if the char sequence is empty.
+ *
+ * @sample samples.text.Strings.last
```

Comment:
This is a generated file, please change the template as well.

Reply:
done